.github/workflows/release-artifacts.yml

```
@@ -95,28 +95,52 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-20.04, macos-10.15]
+        emulate_aarch64: [false, true]
+        # is_pr is a flag used to exclude certain jobs from the matrix on PRs.
+        # It is not read by the rest of the workflow.
         is_pr:
           - ${{ startsWith(github.ref, 'refs/pull/') }}
 
         exclude:
           # Don't build macos wheels on PR CI.
           - is_pr: true
             os: "macos-10.15"
+          # Don't build aarch64 wheels on mac.
+          - os: "macos-10.15"
+            emulate_aarch64: true
+          # Don't build aarch64 wheels on PR CI.
+          - is_pr: true
+            emulate_aarch64: true
 
     steps:
+      - name: Debug
+        run: echo '${{ toJSON(matrix) }}'
+
       - uses: actions/checkout@v3
 
       - uses: actions/setup-python@v3
 
+      - name: Set up QEMU to emulate arm64
+        if: matrix.emulate_aarch64
+        uses: docker/setup-qemu-action@v2
+        with:
+          platforms: arm64
+
       - name: Install cibuildwheel
         run: python -m pip install cibuildwheel==2.9.0 poetry==1.2.0
 
       # Only build a single wheel in CI.
-      - name: Set env vars.
+      - name: "Env: build single wheel for CI"
         run: |
           echo "CIBW_BUILD="cp37-manylinux_x86_64"" >> $GITHUB_ENV
         if: startsWith(github.ref, 'refs/pull/')
 
+      # Only build a single wheel in CI.
+      - name: "Env: build aarch64 wheels"
+        run: |
+          echo "CIBW_ARCHS_LINUX=aarch64" >> $GITHUB_ENV
```

@DMRobertson 
The build failed since x86 and aarch64 flags seems to have been mixed. Did you try to replace `CIBW_BUILD` accordingly?
```suggestion
          echo "CIBW_ARCHS_LINUX=aarch64" >> $GITHUB_ENV
          sed -i '/^CIBW_BUILD=/c\CIBW_BUILD="pp37-manylinux_aarch64"' $GITHUB_ENV
```